core/pom.xml

```diff
@@ -220,6 +220,12 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>javax.mail</groupId>
```

Comment:
I'm assuming this is ok, because it was already done in https://github.com/pentaho/pentaho-analyzer/blob/8e1e19611895b59f649875d4c350f65af96ba5ad/server/pom.xml#L257.

But JavaMail API is a separate library in Java SE (http://www.oracle.com/technetwork/java/javamail/index.html), and I only found it included for OSGi (`karaf/system/javax/mail/mail/1.4.4/mail-1.4.4.jar`). Probably it is Tomcat, being an EE container, that provides its implementation for the non-OSGi.